protocol/pom.xml

```diff
@@ -57,7 +57,7 @@
     <dependency>
       <groupId>org.fidoalliance.fdo</groupId>
       <artifactId>logging-utils</artifactId>
-      <version>1.0-SNAPSHOT</version>
+      <version>${fdo.version}</version>
```

Comment:
This alone seems to be different from the rest. Hope it is intended that way to pick the value.

Reply:
This was a left over from earlier changes. The source change is appropriate.